config/locales/en.yml

```diff
@@ -46,6 +46,7 @@ en:
           attributes:
             title:
               blank: 'Enter your title'
+              too_long: 'Title must be %{count} characters or less'
```

Comment:
Or fewer. Could we fix the other less/fewers in this file at the same time?

Reply:
The error messages are following the GOV.UK Design System guidance on a text input if an input is too long (see https://design-system.service.gov.uk/components/text-input/#if-the-input-is-too-long) and it suggests `‘[whatever it is] must be [number] characters or less’.` for it. 😬 🤷‍♀ 

Do you think we should leave it or change it?

Reply:
"Fewer" is correct. Going to raise a PR on those docs. ⚔️

Reply:
I've updated this in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/159/commits/ee307a286f9238ff416e2ccab6f8d2d6b3c04849.